tests/tape/test_qscript.py

```diff
@@ -1425,7 +1423,7 @@ def test_sample_real_eigvals(self):
     def test_sample_real_and_int_eigvals(self):
         """Test that the tape can correctly determine the output domain for
         multiple sampling measurements with a Hermitian observable with real
-        eigenvalues and another one with integer eigenvalues."""
+        eigenvalues and another sample with integer values."""
         dev = qml.device("default.qubit", wires=3, shots=5)
 
         arr = np.array(
```

Comment:
If you don't mind, would you remove the trailing comma in `arr` below, so it can live in one line? :upside_down_face:

Reply:
done